src/Hit/Git.hs

```diff
@@ -43,18 +43,21 @@ runNew issueNum = do
             "git" ["checkout", "-b", branchName]
 
 -- | @hit commit@ command.
-runCommit :: Text -> IO ()
-runCommit (T.strip -> msg)
+runCommit :: Text -> Bool -> IO ()
+runCommit (T.strip -> msg) (not -> hasIssue)
```

Comment:
Need to be careful with this.

As the option we could use `flag False True` instead of `switch` in the parser, so we don't need to make `not` in here. But I'm okay with this solution!